### PR TITLE
PR-7: Added GCD documentation files

### PR DESCRIPTION
**Pull request by tzulberti-jampp**

### CODE REVIEW

File: docs/installation.rst [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+
+Stable release
+==============
+
+To install gcd, run this command in your terminal:
+
+.. code-block:: console
+
+    pip install gcd
```
Comment (diegobatt) [inline docs/installation.rst]: I believe that this is not true since there is another gcd in pypi

File: docs/installation.rst [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+This is the preferred method to install gcd, as it will always install the most recent
+stable release.
+
+
+From sources
+============
+
+The sources for gcd can be downloaded from the `Github repo`_.
+
+You can either clone the public repository:
```
Comment (danielfujii-jampp) [inline docs/installation.rst]: either? You're not giving another options.

**Comment by tzulberti-jampp** [comment]
> Would this be the PR to include a LICENSE in the repository? If so, let's choose a standard one, and go with it. Waiting for Carlos to choose one is like waiting for hell to freese over.

Added BSD-3 Licence

File: docs/installation.rst (hunk 1 above)
Comment (tzulberti-jampp) [reply docs/installation.rst]: GCD is an open source project, and the github repo isn't private. So I don't want to link the tickets  system we use at Jampp

File: Makefile [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+	find . -name '*.egg-info' -exec rm -fr {} +
+	find . -name '*.egg' -exec rm -f {} +
+
+clean-pyc: ## remove Python file artifacts
+	find . -name '*.pyc' -exec rm -f {} +
+	find . -name '*.pyo' -exec rm -f {} +
+	find . -name '*~' -exec rm -f {} +
+	find . -name '__pycache__' -exec rm -fr {} +
+
+clean-test: ## remove test and coverage artifacts
```
Comment (gonzalezzfelipe) [inline Makefile]: Sory, I just realized another thing. Why was the `test` option removed from the makefile?

Comment (tzulberti-jampp) [reply Makefile]: The test file uses a private docker image in Jampp,. Again this project is open source.

File: docs/installation.rst (hunk 1 above)
Comment (tzulberti-jampp) [reply docs/installation.rst]: Sorry, mixed comments

File: CHANGELOG.rst [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+When major components get significant changes worthy of mention, they
+can be described in a Major section.
+
+v1.1.0 - Unreleased
+===================
+
+Added
+-----
+
+* Created the documentation
```
[the remark this answers is not in the document]
Comment (tzulberti-jampp) [reply CHANGELOG.rst]: GCD is an open source project, and the github repo isn't private. So I don't want to link the tickets system we use at Jampp

**Pull request merged**